pkg/monitor/cluster/reporting.go

```diff
@@ -0,0 +1,42 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strconv"
+)
+
+const (
+	masterRoleLabel = "node-role.kubernetes.io/master"
+	workerRoleLabel = "node-role.kubernetes.io/worker"
+)
+
+// emitReportingMetric emits joined metric to be able to report better on
+// all clusters state in single dashboard
+func (mon *Monitor) emitReportingMetric(ctx context.Context) error {
+	// daily only
+	if mon.dailyRun {
+		var masterCount, workerCount int
+		for _, node := range mon.cache.ns.Items {
+			if _, ok := node.Labels[masterRoleLabel]; ok {
+				masterCount++
+			}
+			if _, ok := node.Labels[workerRoleLabel]; ok {
+				workerCount++
+			}
+		}
+
+		mon.emitGauge("cluster.summary", 1, map[string]string{
```

Comment:
This metric contains half of the things from `cluster.versions`. Maybe we should look into getting rid of `cluster.versions`?

Reply:
how about
1. once this is in all regions in prod, remove the dashboard for versions
2. then PR to remove the cluster.versions

Reply:
Might be a better approach as otherwise we gonna lose data for a bit .
TODO: Raise follow-up item before merge